pom.xml

```diff
@@ -155,7 +155,7 @@
         <maxmind.version>0.8.1</maxmind.version>
         <maxmind-db.version>0.3.4</maxmind-db.version>
         <jna.version>4.0.0</jna.version>
-        <winrm4j.version>0.3.5</winrm4j.version>
+        <winrm4j.version>0.4.0-SNAPSHOT</winrm4j.version>
```

Comment:
jenkins failure

`[WARNING] The POM for io.cloudsoft.windows:winrm4j:jar:0.4.0-SNAPSHOT is missing, no dependency information available`

let's get `0.4.0` out (not snapshot) and simply reference it ?